observability/monitoring/monitoring-with-gitops-dashboards.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+[id="monitoring-with-gitops-dashboards"]
+include::_attributes/common-attributes.adoc[]
+:context: monitoring-with-gitops-dashboards
+= Monitoring with {gitops-shortname} dashboards
+
+toc::[]
+
+You can access a graphical view of {gitops-shortname} instances with {gitops-title} monitoring dashboards to observe the behavior and usage of each instance across the cluster. 
+
+There are three {gitops-shortname} dashboards available:
+
+* *{gitops-shortname} Overview*: See an overview of all {gitops-shortname} instances installed on the cluster, including the number of applications, health and sync status, application and sync activity.
+* *{gitops-shortname} Components*: View detailed information, such as cpu or memory, for an application-controller, repo-server, server, and other {gitops-shortname} components.
+* *{gitops-shortname} gRPC Services*: View metrics related to gRPC service activity between the various components in {gitops-title}.
+
+include::modules/gitops-accessing-gitops-monitoring-dashboards.adoc[leveloffset=+1]
+
+//[role="_additional-resources"]
+//.Additional resources
+//* <add-the-link-to-RHDEVDOCS-5560-once-created>
```

Comment:
The link to the installation will be added with this Jira: https://issues.redhat.com/browse/RHDEVDOCS-5560